Apps/Playground/Android/app/build.gradle

```diff
@@ -114,6 +88,8 @@ tasks.configureEach { task ->
     }
 }
 
+preBuild.dependsOn ':BabylonNative:assembleRelease'
```

Comment:
Does this mean we always use a release build of BabylonNative? If so, I would think this would impact debuggability.

Reply:
I could attach the debugger and step into the code, watch variables,....